### PR TITLE
Raidboss: DR nm and Savage fr

### PR DESCRIPTION
**Pull request by VanSlanzar**

### CODE REVIEW

**Comment by VanSlanzar** [comment]
1 failing

  1) timeline test
       ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.txt
         should not be missing translations:
     AssertionError: ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.js:locale fr:no translation for replaceText `'--stunned--`'
      at Context.<anonymous> (file:///home/runner/work/cactbot/cactbot/test/helper/test_timeline.js:237:17)
      at processImmediate (internal/timers.js:461:21)



npm ERR! Test failed.  See above for more details.
Error: Process completed with exit code 1.

I not understand, that not exist in timeline `drs.txt`.
I made a mistake ? @quisquous

**Comment by VanSlanzar** [comment]
- I revert, because i'm not sur for line break.
- There is this as well.
fail

**Comment by quisquous** [comment]
> I not understand, `--stunned--` that not exist in timeline `drs.txt`.

I just added this recently: https://github.com/quisquous/cactbot/blob/main/ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.txt#L529

> * I revert, because i'm not sur for line break.
> * There is this as well.
>   fail

Sorry, I'm not sure what it is going on here.  I'll take a look

File: ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.js [hunk 1]
```diff
@@ -939,10 +971,12 @@ export default {
         firstWeave: {
           en: 'Go North (donut bottom/circle top)',
           de: 'Geh nach Norden (Donut unten/Kreise oben)',
+          fr: 'Allez au nord (donut bas/cercle haut)',
         },
         secondWeave: {
           en: 'Stay South (square bottom/circle top)',
           de: 'Geh nach Süden (Viereck unten/Kreise oben)',
+          fr: 'Restez au sud (fond carré/cercle haut)'
```
Comment (quisquous) [inline ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.js:979]: Maybe it's confused, but I think this is the missing comma.

Comment (VanSlanzar) [reply ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.js]: I have however checked several times. Thank you and sorry for the unnecessary inconvenience.
Just a quick note, all local **npm run tests were successful**. I am troubled that I had all of these not found errors. Is the command `npm run test` still?

**Pull request merged**